Engines/Wine/QuickScript/Quick Script/script.js

```diff
@@ -76,6 +76,27 @@ QuickScript.prototype.executable = function (executable, args) {
     return this;
 };
 
+/**
+ * set environment
+ * @param {string} environment variables
+ * @returns {QuickScript} QuickScript object
+ */
+QuickScript.prototype.environment = function (environment) {
```

Comment:
Do you perhaps want to refactor our quick script classes similar how I changed ShortcutReader in #1008?

Reply:
No idea ^^.